lib/hooks.js

```diff
@@ -15,6 +15,22 @@ exports.onResponse = function (request, reply, next) {
     next();
 };
 
+exports.onRequestSize = function (request, reply, next) {
+    const contentLength = request.headers?.['content-length'];
```

Comment:
We need to remove this, because not supported in node 12

Reply:
ThatThis is already been done :+1: